contrib/google.golang.org/grpc/option.go

```diff
@@ -49,3 +50,12 @@ func WithStreamMessages(enabled bool) InterceptorOption {
 		cfg.traceStreamMessages = enabled
 	}
 }
+
+// NoDebugStack disables backtrace generation for any type of errors.
```

Comment:
Let's not use the word "backtrace" anywhere please. It has no meaning in Go. Please use the following:
```go
// NoDebugStack disables debug stacks for traces with errors. This is useful in situations
// where errors are frequent and the overhead of calling debug.Stack may affect performance.
```
I'd appreciate it if you would follow this exact wording.

Reply:
I don't mind though in CS backtrace or stacktrace refer to a [trace of the function calls made by a program](https://en.wiktionary.org/wiki/backtrace). It's a common name for a [function](https://linux.die.net/man/3/backtrace) or [module](https://docs.rs/backtrace/0.3.9/backtrace/) .